src/auto/injector.js

```diff
@@ -359,6 +359,8 @@ function annotate(fn, strictDi, name) {
  * * {@link auto.$provide#service service(class)} - registers a **constructor function**, `class`
  *     that will be wrapped in a **service provider** object, whose `$get` property will instantiate
  *      a new object using the given constructor function.
+ * * {@link auto.$provide#decorator decorator(name)} - registers a **service decorator** with the
```

Comment:
service decorator is not correct, as not only services can be decorated. You should call it **decorator function**. In the function signature, the decorator fn is also missing: decorator(name, decoratorFn)